codes/requirements.laxed.txt

```diff
@@ -16,6 +16,9 @@ einops
 lambda-networks
 mup
 
+#UI
+customtkinter
+
```

Comment:
i will probably separate this out to a 2nd file for windows specifically, or add some [windows] package installation tag

Reply:
ctk isn't windows only

Reply:
Oh, my apologies.

Is the `Setup DLAS.bat` script supposed to be run on windows only, then? Wouldn't the venv be missing? Or are linux users okay with the normal installation approach?

Reply:
Well this is just for windows currently, I dont have Linux installed currently to debug an SH file